force-app/main/default/classes/ProgramEngagementRollupsSchedulable_TEST.cls

```diff
@@ -9,57 +9,79 @@
 
 @isTest
 private class ProgramEngagementRollupsSchedulable_TEST {
-    private static BasicStub attendanceServiceStub = new BasicStub(
-        ServiceDeliveryService.class
-    );
-
     @testSetup
     static void setup() {
         TestUtil.turnOffFeatureGates();
         TestDataFactory.generateAttendanceData('Monthly');
     }
 
     @IsTest
-    private static void shouldCallServiceWithProgramEngagements() {
-        // List<String> attendanceStatuses = new ServiceDeliveryService().attendanceStatuses;
+    private static void shouldQueryProgramEngagementsAndExecuteBatch() {
+        List<ProgramEngagement__c> programEngagementsWithRollupData = [
+            SELECT Id
+            FROM ProgramEngagement__c
+            WHERE
+                NumPresentServiceDeliveries__c >= 0
+                OR NumAbsentServiceDeliveries__c >= 0
+                OR LastServiceDate__c != NULL
+                OR ConsecutiveAbsences__c >= 0
+        ];
+
+        System.assert(
+            !programEngagementsWithRollupData.isEmpty(),
+            'Expected test data to be queried.'
+        );
+
+        Test.startTest();
+
+        ProgramEngagementRollupsSchedulable programEngagementRollups = new ProgramEngagementRollupsSchedulable();
+        Database.executeBatch(programEngagementRollups);
+
+        Test.stopTest();
+    }
+
+    @IsTest
+    private static void shouldScheduleProgramEngagementRollupsBatch() {
+        List<ProgramEngagement__c> programEngagementsWithRollupData = [
+            SELECT Id
+            FROM ProgramEngagement__c
+            WHERE
+                NumPresentServiceDeliveries__c >= 0
+                OR NumAbsentServiceDeliveries__c >= 0
+                OR LastServiceDate__c != NULL
+                OR ConsecutiveAbsences__c >= 0
+        ];
+
+        System.assert(
+            !programEngagementsWithRollupData.isEmpty(),
+            'Expected test data to be queried.'
+        );
+
+        Test.startTest();
+
+        String hour = String.valueOf(Datetime.now().hour());
```

Comment:
I wonder if we can build this in the test data factory so we don't need to rebuild for each test?